types/lodash/tsconfig.json

```diff
@@ -19,6 +19,391 @@
     },
     "files": [
         "index.d.ts",
-        "lodash-tests.ts"
+        "lodash-tests.ts",
+        "fp/add.d.ts",
```

Comment:
These file references were recently removed. I'm not sure if they're actually needed, would like to hear from the TypeScript team on this. FYI there is a PR in progress to stop auto-generating these.

Reply:
I'm going to leave only the index.d.ts file here as it seems to be breaking the CI process too 😌